recipes/drupal8-multisite/README.md

```diff
@@ -124,40 +101,40 @@ And reruning `drush status` inside the `sites/umami` directory inside the *web*
 
 ## Set up drush site aliases
 
-[Drush site aliases](http://docs.drush.org/en/9.x/usage/#site-aliases) help to address individual sites without being in the specific directory inside the web container. There is an example *self.site.yml* already with `drush/sites` folder, showing the basic format for default (self) site with environments *prod* and *stage*.
+[Drush site aliases](http://docs.drush.org/en/9.x/usage/#site-aliases) help to address individual sites without being in the specific directory inside the web container.
 
 We'll set the *root* and *uri* parameters to point to the Drupal root (within container) and the full url to the site instance.
 
-1. `cd drush/sites`
-2. Copy the aliases files into your project's drush/sites directory from this [drush/sites](drush/sites) directory
-3. Run `ddev drush site:alias` from outside the container to check that the alias is recognized
-4. Run `ddev drush @umami.ddev status` to check that all site-specific information is now present, including full *Site URI*
-5. Run `ddev drush @basic.ddev status` to check that the alias is working for the second site too.
-6. Run `ddev drush status` to see what the default configuration still shows.
+1. `mkdir -p drush/sites && cd drush/sites`
+2. Inspect and copy the sample aliases files in [drush/sites](drush/sites) into your project's drush/sites directory.
+3. Run `ddev exec drush site:alias` from outside the container to check that the alias is recognized
+4. Run `ddev exec drush @umami.ddev status` to check that all site-specific information is now present, including full *Site URI*
+5. Run `ddev exec drush @basic.ddev status` to check that the alias is working for the second site too.
+6. Run `ddev exec drush status` to see what the default configuration still shows.
 
 ## Add more sites
 
 Now that we have a basic setup in place, new site requires us to:
 
 1. Update ddev's [config.multisite.yaml](dot.ddev/config.multisite.yaml) to add the database and set the additional_hostnames (remember to run `ddev restart` to pick up the changes). This can also be done directly in the .ddev/config.yaml.
 2. Create a directory for the new site in `web/sites/<newsitename>`.
-3. Copy `settings.php` to the new directory and updating the database name.
+3. Copy `settings.php` to the new directory and update the database name.
 4. Add new alias to the `sites.php`.
 5. Copy a drush alias and updating the `uri` option.
 6. Visit each new site to install Drupal.
 
 ## Conclusion
 
-Of course much more work lies ahead.  The `settings.php` file alone may need a lot more additional configuration. This however, is a standard Drupal issue covered in the documentation.
+Of course much more work lies ahead.  The `settings.php` file alone may need a lot more additional configuration. This however, is a standard Drupal issue covered in Drupal's documentation.
```

Comment:
IMO we can remove this sentence, it doesn't help understand, and just gives a feeling you did something wrong.